src/mixins/socket.js

```diff
@@ -96,8 +96,12 @@ export default {
                 wsHost = protocol + location.host;
             }
 
+            // always starts and ends with '/'
+            const basePath = document.querySelector("head base").getAttribute("href");
+
             socket = io(wsHost, {
-                transports: [ "websocket" ],
+                transports: ["websocket"],
```

Comment:
```suggestion
                transports: [ "websocket" ],
```

Reply:
I wonder how that happened, since I prefer the original style...

Reply:
We are using ESLint: https://github.com/louislam/uptime-kuma/blob/master/.eslintrc.js

Reply:
I mean I wonder how did the change from spaced to non-spaced happen, since I would not do it myself. I would leave it spaced.